docs/source/client.rst

```diff
@@ -30,14 +30,18 @@ Below are examples of endpoints and how they map to API Client methods:
 +-----------------------------------+-------------------------------------------+
 | ``GET /workflows/1/executions``   | ``client.workflows.list_executions(1)``   |
 +-----------------------------------+-------------------------------------------+
-| ``PATCH /workflows/1``            | ``client.workflows.patch(1)``             |
+| ``PATCH /workflows/1``            | ``client.workflows.patch(1, ...)``        |
 +-----------------------------------+-------------------------------------------+
 | ``POST /workflows/1/executions``  | ``client.workflows.post_executions(1)``   |
 +-----------------------------------+-------------------------------------------+
 | ``GET /workflows/1/executions/2`` | ``client.workflows.get_executions(1, 2)`` |
 +-----------------------------------+-------------------------------------------+
-| ``DELETE /workflows/1``           | ``client.workflows.delete(1)``            |
-+-----------------------------------+-------------------------------------------+
+
+Note that Python's built-in ``help`` function can be used to see lists of
```

Comment:
sorry @aripollak, could you take one more glance when you have a moment? 🤦 

I just built and checked the docs locally and was reminded that rst is not the same as markdown in that it uses double backticks for code.  i really prefer markdown...